pyam/core.py

```diff
@@ -380,7 +385,17 @@ def timeseries(self, iamc_index=False):
         iamc_index: bool, default False
             if True, use `['model', 'scenario', 'region', 'variable', 'unit']`;
             else, use all `data` columns
+
+        Raises
+        ------
+        AttributeError
+            ``IamDataFrame`` is empty
+        ValueError
+            reducing to IAMC-index yields an index with duplicates
         """
+        if self.empty:
+            raise AttributeError('this `IamDataFrame` is empty')
```

Comment:
Should this be an attribute error or perhaps a value error instead?

Reply:
done